src/test/regress/expected/citus_local_tables.out

```diff
@@ -559,7 +559,7 @@ FROM pg_dist_partition WHERE logicalrelid = 'citus_local_table_4'::regclass;
 (1 row)
 
 SELECT column_name_to_column('citus_local_table_4', 'a');
-                                                 column_name_to_column
+                                                  column_name_to_column
```

Comment:
I wonder the reason for this change

Reply:
I am not sure. As long as the tests pass, I think everything is ok